CRM/Mailing/xml/Menu/Mailing.xml

```diff
@@ -40,6 +40,15 @@
     <path>civicrm/admin/mailSettings</path>
     <title>Mail Accounts</title>
     <page_callback>CRM_Admin_Page_MailSettings</page_callback>
+    <desc>List email accounts.</desc>
+    <access_arguments>access CiviCRM,access CiviMail</access_arguments>
+    <adminGroup>CiviMail</adminGroup>
+    <weight>420</weight>
+  </item>
+  <item>
+    <path>civicrm/admin/mailSettings/edit</path>
+    <title>Mail Accounts</title>
+    <page_callback>CRM_Admin_Page_MailSettings</page_callback>
```

Comment:
Can this point directly to `CRM_Admin_Form_MailSettings` - I think that ought to work and it's a nice step toward detangling the form from the page.